.gitignore

```diff
@@ -5,13 +5,3 @@ nbproject/
 *.swp
 build/
 
-# Generated files and directories.
-
-**/*odb\.cxx
```

Comment:
These lines for ODB remained here due to a rebase missing? This patch in itself did not touch ODB-related stuff.

Reply:
Yes, those should have been cleaned up in #323.